<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>factory</class>
 <widget class="QWidget" name="factory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>0</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>FACTORY MENU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>61</y>
     <width>200</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_alignment">
      <property name="minimumSize">
       <size>
        <width>108</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>ALIGNMENT</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_factoryDefault">
      <property name="minimumSize">
       <size>
        <width>108</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>FACTORY DEFAULT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_radarParams">
      <property name="minimumSize">
       <size>
        <width>108</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>RADAR PARAMS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_factoryCalibrate">
      <property name="minimumSize">
       <size>
        <width>108</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>CALIBRATE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
